includes/class-wcpdf-settings-callbacks.php

```diff
@@ -290,38 +288,38 @@ public function radio_button( $args ) {
 	public function multiple_text_input( $args ) {
 		extract( $this->normalize_settings_args( $args ) );
 
-		if (!empty($header)) {
-			echo "<p><strong>{$header}</strong>:</p>";
+		if ( ! empty( $header ) ) {
+			echo wp_kses_post( "<p><strong>{$header}</strong>:</p>" );
 		}
 
-		printf('<p class="%s multiple-text-input">', $id);
+		printf('<p class="%s multiple-text-input">', esc_attr( $id ) );
 		foreach ($fields as $name => $field) {
 			$size = $field['size'];
 			$placeholder = isset( $field['placeholder'] ) ? $field['placeholder'] : '';
```

Comment:
`! empty()`?

Reply:
`$field` exists so `isset()` is safe here